c/tskit/tables.c

```diff
@@ -2990,7 +2901,7 @@ tsk_mutation_table_expand_metadata(tsk_mutation_table_t *self, size_t additional
         = (tsk_size_t) TSK_MAX(additional_length, self->max_metadata_length_increment);
     tsk_size_t new_size = self->max_metadata_length + increment;
 
-    if (check_offset_overflow(self->metadata_length, increment)) {
```

Comment:
Shouldn't this be expand_ragged_column also?

Reply:
Whoops, bot sure how I missed that one. Have searched through again to check the others.